### PR TITLE
update support

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

**Comment by yapei** [comment]
@liangxia Could you help review and approve if no objections?

File: lib/openshift/subscription.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ def startingCSV(user: nil, quiet: false, cached: false)
       return rr.dig('status', 'currentCSV')      
     end
 
+    def installedCSV(user: nil, quiet: false, cached: false)
```
Comment (liangxia) [inline lib/openshift/subscription.rb:15]: Perhaps make quiet/cached defaults to true ?

Comment (yapei) [reply lib/openshift/subscription.rb:15]: @liangxia good point, I agree `quiet` should be `true` by default, however I'm not sure if `cached` should be true or false by default, anyway let me update all to `true` by default and see if any issues

**Pull request merged**